src/plugins/unified_search/public/dataview_picker/dataview_list_popover.test.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { shallow } from 'enzyme';
+import React from 'react';
+import type { IStorageWrapper } from '@kbn/kibana-utils-plugin/public';
+import { Storage } from '@kbn/kibana-utils-plugin/public';
+import { StubBrowserStorage } from '@kbn/test-jest-helpers';
+import type { DataViewListItemEnhanced } from './dataview_list';
+import { SortingService } from './sorting_service';
+
+import { DataViewsListPopover } from './dataview_list_popover';
+
+describe('DataViewsListPopover', () => {
+  let storage: IStorageWrapper;
+  let sortingService: SortingService<DataViewListItemEnhanced>;
+
+  beforeEach(() => {
+    storage = new Storage(new StubBrowserStorage());
+    sortingService = new SortingService<DataViewListItemEnhanced>(
+      {
+        alphabetically: (item) => item.name ?? item.title,
+      },
+      storage
+    );
+  });
+  it('should render `EuiPopover`', () => {
+    const wrapper = shallow(
+      <DataViewsListPopover sortingService={sortingService} handleSortingChange={jest.fn()} />
+    );
+    expect(wrapper).toMatchInlineSnapshot(`
```

Comment:
I am not a fan of the snapshot testing. Can we add more specific tests to check the sorting options are rendered correctly and the on Change callbacks?

Reply:
yes, sure. I now was engaged in addition of new tests. didn't have time to push